.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--
+
+ PLEASE MAKE SURE NO SENSIBLE INFORMATION ABOUT YOUR
+ SYSTEM IS EXPOSED WHEN SUBMITTING FEEDBACK!
+
+ Thank you for considering giving feedback to the project!
+
+ When adding your feedback/bug please keep in mind:
+   - This is not a support channel.
```

Comment:
`This is not a support channel.` -> `This is not a support channel. If you are having payed support from SUSE business - please go to ...`

Reply:
s/payed/paid/